.github/workflows/windows-macos.yml

```diff
@@ -275,11 +275,51 @@ jobs:
       if: ${{ runner.os == 'macOS' }}
       run: ccache --show-stats --zero-stats
 
+    - name: Enable coredump
```

Comment:
This same workflow file gets used for producing release builds. Should we constrain this action to only certain branches?

Reply:
I think the real codesign would overwrite the changes made here anyway, wouldn't that be the case?